registration/backends/default/views.py

```diff
@@ -139,3 +140,18 @@ def activate(self, *args, **kwargs):
 
     def get_success_url(self, user):
         return ('registration_activation_complete', (), {})
+
+
+class ResendActivationView(BaseResendActivationView):
+    def resend_activation(self, *args, **kwargs):
+        """
+        Given an email, look up user by email and resend activation key if user
+        is not already activated or previous activation key has not expired.
+        """
+        site = get_current_site(self.request)
+        email = kwargs.get('email', '')
```

Comment:
I think that makes sense, albeit following the code is not super clear. I'm not sure of a simpler implementation though.